zipline/__main__.py

```diff
@@ -6,8 +6,12 @@
 import pandas as pd
 from six import text_type
 
-from zipline.utils.compat import wraps
 from zipline.data import bundles as bundles_module
+from zipline.utils.calendars.calendar_utils import (
+    get_calendar,
+    _default_calendar_factories
```

Comment:
We probably don't want to keep the leading `_` if we're importing this here.

Better yet, what would you think of including a line in `calendar_utils` like
```
default_calendar_names = _default_calendar_factories.keys()
```
and importing that instead?

Reply:
Sounds good to me. I also wrapped it with `sorted()`